src/v/kafka/server/handlers/fetch.cc

```diff
@@ -230,14 +230,18 @@ make_ntp_fetch_config(const model::ntp& ntp, const fetch_config& fetch_cfg) {
 }
 
 ss::future<read_result> read_from_ntp(
-  cluster::partition_manager& pm,
-  cluster::metadata_cache& md_cache,
+  cluster::partition_manager& cluster_pm,
+  coproc::partition_manager& coproc_pm,
```

Comment:
what if coproc partition manager is stopped before finishing processing this fetch request ?

Reply:
I modified its shutdown to shutdown when `cluster::partition_manager` shuts down, since the semantics are the same, is this good enough?